docs/connectors/getting-started-connectors/README.adoc

```diff
@@ -123,65 +125,52 @@ image::connectors-diagram.png[Illustration of data flow from data source through
 endif::[]
 
 [id="proc-configuring-kafka-for-connectors_{context}"]
-== Configuring the {product-kafka} instance for use with {product-long-connectors}
+== Verifying that you have the prerequisites for using {product-long-connectors}
 
 [role="_abstract"]
-After you create a {product-kafka} instance, configure it for use with {product-long-connectors} by performing the following tasks:
-
-* Create *Kafka topics* to store messages sent by producers (data sources) and make them available to consumers (data sinks).
-* Create *service accounts* that allow you to connect and authenticate your Connectors with Kafka instances.
-* Set up *access rules* for the service accounts that define how your Connectors can access and use the associated Kafka instance topics.
-
-The number of Kafka topics and service accounts that you create, and the access rules for the service accounts, depend on your application.
-
-For this example, you create one Kafka topic, named *test*, one service account, and you define access for the service account.
-
 ifdef::qs[]
-.Prerequisites
-* You've created a {product-kafka} instance and the instance is in the *Ready* state.
-endif::[]
+Before you can use {product-connectors}, you must complete the link:https://console.redhat.com/application-services/learning-resources?quickstart=getting-started[Getting started with {product-long-kafka}] quick start to set up the following components:
 
+* A *Kafka instance* that you can use for {product-connectors}.
+* A *Kafka topic* to store messages sent by data sources and make the messages available to data sinks.
+* A *service account* that allows you to connect and authenticate your {connectors} instances with your Kafka instance.
+* *Access rules* for the service account that defines how your {connectors} instances can access and use the topics in your Kafka instance.
+endif::[]
 ifndef::qs[]
```

Comment:
Do you need the qs conditionalization in the Prerequisites section? I don't think any of the content here will ever show up in the actual quick start in the HCC. For the actual quick start, the prereqs are already defined in quickstart.yml (https://github.com/redhat-developer/app-services-guides/blob/d655bc3c580ea7fbbf02468c427111e8e7202ce7/docs/connectors/getting-started-connectors/quickstart.yml#L17), and the "Prerequisites" section is not referenced anywhere in quickstart.yml.

Reply:
Unfortunately, I do need the prereq conditionalization in the Prerequisties section in order for it to render correctly in both places - at least according to the local builds that I did.